From b610503103dba0475247c292a61393926a09085c Mon Sep 17 00:00:00 2001
From: Dmitry Kazakov <dimula73@gmail.com>
Date: Fri, 18 Apr 2025 11:40:20 +0200
Subject: [PATCH] Fix version file generation in version.py

When VERSION file exists, it overrides the revision deduced from git tags
---
 libavutil/version.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libavutil/version.py b/libavutil/version.py
index 8b133f0860..0c4f6fcac4 100644
--- a/libavutil/version.py
+++ b/libavutil/version.py
@@ -92,8 +92,8 @@ if __name__=='__main__':
                     old_revision = parts[2].split('"')[1]
                     break
 
-    if revision != old_revision:
+    if version != old_revision:
         with open(args.output, 'w') as f:
             f.write('/* Automatically generated by version.py, do not manually edit! */\n')
             f.write('#pragma once\n')
-            f.write('#define FFMPEG_VERSION "%s"\n' % revision)
+            f.write('#define FFMPEG_VERSION "%s"\n' % version)
-- 
2.25.1
